web-application/simple-calculator/script.js

```diff
@@ -0,0 +1,14 @@
+function buttonclick(val)
+{
+    console.log(val)
+    document.getElementById("screen").value=document.getElementById("screen").value+=val;
+}
+
+function cleardisplay(){
+    document.getElementById("screen").value=""
+}
+function equalclick(){
+    var text=document.getElementById("screen").value
+    var result=eval(text)

```

Comment:
Avoid using `eval` in unsecured way.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!

Reply:
Removed eval()